### PR TITLE
Fix aria-label of Sendbox

### PR DESCRIPTION
**Pull request by corinagum**
Fixes https://github.com/microsoft/BotFramework-Emulator/issues/1823

## Changelog Entry

-  Fix [Emulator1823](https://github.com/microsoft/BotFramework-Emulator/issues/1823). Fix Sendbox "Type your message" being read twice by AT, by [@corinagum](https://github.com/corinagum) in PR [#2423](https://github.com/microsoft/BotFramework-WebChat/pull/2423)

## Description

This is an accessibility fix for the sendbox in Web Chat, where the narrator was reading the placeholder twice. Now it will say, "Sendbox edit 'Type your message'" instead of "Type your message edit type your message"

### CODE REVIEW

File: packages/component/src/SendBox/TextBox.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ const TextBox = ({ className, disabled, language, onChange, onKeyPress, onSubmit
           {({ sendFocusRef }) =>
             !sendBoxTextWrap ? (
               <input
-                aria-label={typeYourMessageString}
+                aria-label={'Sendbox'}
```
Comment (tdurnford) [inline packages/component/src/SendBox/TextBox.js]: Does this need to be localized?

Comment (compulim) [reply packages/component/src/SendBox/TextBox.js]: And we should use `aria-label="Sendbox"` rather than `aria-label={'Sendbox'}` for strings.

Comment (corinagum) [reply packages/component/src/SendBox/TextBox.js]: Thanks, localization added (ping)

File: CHANGELOG.md [hunk 2]
```diff
@@ -87,6 +87,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 -  Fix [#2365](https://github.com/microsoft/BotFramework-WebChat/issues/2365). Fix Adaptive Card `pushButton` appearance on Chrome, by [@corinagum](https://github.com/corinagum) in PR [#2382](https://github.com/microsoft/BotFramework-WebChat/pull/2382)
 -  Fix [#2379](https://github.com/microsoft/BotFramework-WebChat/issues/2379). Speech synthesis can be configured off by passing `null`, by [@compulim](https://github.com/compulim) in PR [#2408](https://github.com/microsoft/BotFramework-WebChat/pull/2408)
 -  Fix [#2418](https://github.com/microsoft/BotFramework-WebChat/issues/2418). Connectivity status should not waste-render every 400 ms, by [@compulim](https://github.com/compulim) in PR [#2419](https://github.com/microsoft/BotFramework-WebChat/pull/2419)
+-  Fix [Emulator1823](https://github.com/microsoft/BotFramework-Emulator/issues/1823). Fix Sendbox "Type your message" being read twice by AT, by [@corinagum](https://github.com/corinagum) in PR [#2423](https://github.com/microsoft/BotFramework-WebChat/pull/2423)
```
Comment (compulim) [inline CHANGELOG.md]: `Emulator1823`? 🤣

Comment (tdurnford) [inline CHANGELOG.md]: Might be a little cleaner
```suggestion
-  Fix [Emulator:1823](https://github.com/microsoft/BotFramework-Emulator/issues/1823). Fix Sendbox "Type your message" being read twice by AT, by [@corinagum](https://github.com/corinagum) in PR [#2423](https://github.com/microsoft/BotFramework-WebChat/pull/2423)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -87,6 +87,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 -  Fix [#2365](https://github.com/microsoft/BotFramework-WebChat/issues/2365). Fix Adaptive Card `pushButton` appearance on Chrome, by [@corinagum](https://github.com/corinagum) in PR [#2382](https://github.com/microsoft/BotFramework-WebChat/pull/2382)
 -  Fix [#2379](https://github.com/microsoft/BotFramework-WebChat/issues/2379). Speech synthesis can be configured off by passing `null`, by [@compulim](https://github.com/compulim) in PR [#2408](https://github.com/microsoft/BotFramework-WebChat/pull/2408)
 -  Fix [#2418](https://github.com/microsoft/BotFramework-WebChat/issues/2418). Connectivity status should not waste-render every 400 ms, by [@compulim](https://github.com/compulim) in PR [#2419](https://github.com/microsoft/BotFramework-WebChat/pull/2419)
+-  Fix [Emulator:1823](https://github.com/microsoft/BotFramework-Emulator/issues/1823). Fix Sendbox "Type your message" being read twice by AT, by [@corinagum](https://github.com/corinagum) in PR [#2423](https://github.com/microsoft/BotFramework-WebChat/pull/2423)
```
Comment (tdurnford) [inline CHANGELOG.md]:
```suggestion
-  Fix [Emulator:#1823](https://github.com/microsoft/BotFramework-Emulator/issues/1823). Fix Sendbox "Type your message" being read twice by AT, by [@corinagum](https://github.com/corinagum) in PR [#2423](https://github.com/microsoft/BotFramework-WebChat/pull/2423)
```

**Pull request merged**